cl_dll/ff/ff_hud_deathnotice.cpp

```diff
@@ -264,10 +265,9 @@ void CHudDeathNotice::Paint()
 		}
 		else
 		{
-
-			wchar_t victim[ 256 ];
-			wchar_t killer[ 256 ];
-
+			wchar_t victim[ 256 ];			
+			wchar_t killer[ 512 ]; // bumped this up for kill-assist msgs 

```

Comment:
Yeah, might be overkill. See [MAX_PLAYER_NAME_LENGTH](https://github.com/fortressforever/fortressforever/blob/9e3d589b33c821f16627607d76b63b27d2844caf/public/const.h#L36) which is set to 32 (effectively 31).